openreview/conference/templates/areachairWebfield.js

```diff
@@ -813,7 +813,8 @@ var registerEventHandlers = function() {
     var $currDiv = $('#' + paperNumber + '-add-reviewer');
     var userToAdd = $currDiv.find('input').attr('value_id').trim();
 
-    if (!userToAdd) {
+    if (!userToAdd || !((userToAdd.indexOf('@') > 0) || userToAdd.startsWith('~'))) {
```

Comment:
Are you missing ! before userToAdd.startsWith('~')?

Reply:
This is what I am checking here:
(No input Given) 
OR
NOT ( ( input has the char '@' but does not start at '@' ) OR (input starts with char '~' (e.g. when AC/PC select a name from the dropdown)) )

Reply:
you are right! I missed the parenthesis.